setup.py

```diff
@@ -22,7 +22,7 @@
     os.system("python setup.py sdist bdist_wheel")
     os.system("twine upload dist/*")
     print("You probably want to also tag the version now:")
-    print("  git tag -a %s -m 'version %s'" % (version, version))
+    print("  git tag -a %s -m 'version %s'" % (__version__, __version__))
```

Comment:
It does make sense. It's creating a new tag as the current version with message as the current version.